server/src/controllers/api/users.js

```diff
@@ -137,7 +137,7 @@ class UsersApiController {
 
     User.findOne({
       email: email.trim().toLowerCase()
-    }, '-_id -__v -hashedPassword -donations').exec((err, foundUser) => {
```

Comment:
Quick question, is there a reason why the donations object is included since it was excluded initially ?.

Reply:
Was a mistake. Thanks for pointing that out.

Reply:
Fixed!